services/teamcity/teamcity-build.tester.js

```diff
@@ -24,7 +24,7 @@ t.create('live: codebetter known build')
   .expectJSONTypes(
     Joi.object().keys({
       name: 'build',
-      value: withRegex(buildStatusRegex),
+      value: Joi.equal(buildStatusValues),
```

Comment:
I think this needs a `required()`. Rather than call `Joi.equal` twice, you can call it once at the top.

Reply:
Yea, I just confirmed that here: https://runkit.com/melnikow/joi-schema-testing